### PR TITLE
US 1583733 Add missing language IDs - 01

### PR DESCRIPTION
**Pull request by v-thpra**
[US 1583733](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1583733) - Fix CATS P1 issues by adding missing language IDs to code blocks.

**NOTE:** Most of these changes add the "console" language ID to diagrams of folder structures. Not what the ID's are intended for, but the best match if we must have an ID.

Contributes to #2192

### CODE REVIEW

File: docs/architecture/microservices/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md [hunk 1]
```diff
@@ -401,7 +401,7 @@ Environment variables are created and initialized in different ways, depending o
 
 The following example shows an .env file like the [.env](https://github.com/dotnet-architecture/eShopOnContainers/blob/master/.env) file for the eShopOnContainers application.
 
-```
+```console
```
Comment (mairaw) [inline docs/architecture/microservices/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md]: this is not really console. Maybe change it to env? I believe docs will render anything we put there but it won't colorize for unknown identifiers 

```suggestion
```env
```

File: docs/core/build/distribution-packaging.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ As .NET Core becomes available on more and more platforms, it's useful to learn
 
 When installed, .NET Core consists of several components that are laid out as follows in the filesystem:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/build/distribution-packaging.md]: This one is hard. Maybe we should skip it? It's just showing the file structure in a visual way.

```suggestion
```
```

File: docs/core/deploying/runtime-store.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Starting with .NET Core 2.0, it's possible to package and deploy apps against a
 
 This feature is implemented as a *runtime package store*, which is a directory on disk where packages are stored (typically at */usr/local/share/dotnet/store* on macOS/Linux and *C:/Program Files/dotnet/store* on Windows). Under this directory, there are subdirectories for architectures and [target frameworks](../../standard/frameworks.md). The file layout is similar to the way that [NuGet assets are laid out on disk](/nuget/create-packages/supporting-multiple-target-frameworks#framework-version-folder-structure):
 
-```
+```console
```
Comment (mairaw) [inline docs/core/deploying/runtime-store.md]: ditto 

```suggestion
```
```

File: docs/core/porting/third-party-deps.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ The easiest way to inspect NuGet Package folders is to use the [NuGet Package Ex
 
 Look for a folder with any of the following names:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/porting/third-party-deps.md]: ditto

```suggestion
```
```

File: docs/core/porting/winforms.md [hunk 5]
```diff
@@ -63,7 +63,7 @@ When porting a .NET Framework Windows Forms application, there are a few things
 
 The new .NET Core 3.0 project you create must be in a different directory from your .NET Framework project. If they're both in the same directory, you may run into conflicts with the files that are generated in the **obj** directory. In this example, we'll create a directory named **MyFormsAppCore** in the **SolutionFolder** directory:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/porting/winforms.md]: ditto 

```suggestion
```
```

File: docs/core/porting/winforms.md [hunk 6]
```diff
@@ -93,7 +93,7 @@ dotnet new winforms -o MyFormsAppCore -n MyFormsCore
 
 After you create the **MyFormsCore.csproj**, your directory structure should look like the following:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/porting/winforms.md]: ditto

```suggestion
```
```

File: docs/core/porting/winforms.md [hunk 7]
```diff
@@ -191,7 +191,7 @@ Using the previous step's example, lets expand what projects and files we're wor
 | **MyControlsCore.csproj** | The name of the new .NET Core library project you create. |
 | **MyCoreControls.dll** | The .NET Core Windows Forms Controls library. |
 
-```
+```console
```
Comment (mairaw) [inline docs/core/porting/winforms.md]: ditto 

```suggestion
```
```

File: docs/core/porting/wpf.md [hunk 8]
```diff
@@ -65,7 +65,7 @@ When porting a .NET Framework WPF application, there are a few things you must c
 
 The new .NET Core 3.0 project you create must be in a different directory from your .NET Framework project. If they're both in the same directory, you may run into conflicts with the files that are generated in the **obj** directory. In this example, you'll create a directory named **MyWPFAppCore** in the **SolutionFolder** directory:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/porting/wpf.md]: ditto 

```suggestion
```
```

File: docs/core/porting/wpf.md [hunk 9]
```diff
@@ -95,7 +95,7 @@ dotnet new wpf -o MyWPFAppCore -n MyWPFCore
 
 After you create the **MyWPFCore.csproj**, your directory structure should look like the following:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/porting/wpf.md]: ditto 

```suggestion
```
```

File: docs/core/rid-catalog.md [hunk 10]
```diff
@@ -50,7 +50,7 @@ The above RID specifies that `osx.10.12-x64` imports `osx.10.11-x64`. So, when N
 
 The following example shows a slightly bigger RID graph also defined in the *runtime.json*  file:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/rid-catalog.md]: ditto 

```suggestion
```
```

File: docs/core/testing/unit-testing-fsharp-with-dotnet-test.md [hunk 11]
```diff
@@ -17,7 +17,7 @@ Inside this new directory, run [`dotnet new sln`](../tools/dotnet-new.md) to cre
 makes it easier to manage both the class library and the unit test project.
 Inside the solution directory, create a *MathService* directory. The directory and file structure thus far is shown below:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/testing/unit-testing-fsharp-with-dotnet-test.md]: ditto 

```suggestion
```
```

File: docs/core/testing/unit-testing-fsharp-with-dotnet-test.md [hunk 12]
```diff
@@ -37,7 +37,7 @@ to add the class library project to the solution.
 
 Next, create the *MathService.Tests* directory. The following outline shows the directory structure:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/testing/unit-testing-fsharp-with-dotnet-test.md]: ditto 

```suggestion
```
```

File: docs/core/testing/unit-testing-fsharp-with-dotnet-test.md [hunk 13]
```diff
@@ -58,15 +58,15 @@ Make the *MathService.Tests* directory the current directory and create a new pr
...
 dotnet add reference ../MathService/MathService.fsproj
 ```
 
 You can see the entire file in the [samples repository](https://github.com/dotnet/samples/blob/master/core/getting-started/unit-testing-with-fsharp/MathService.Tests/MathService.Tests.fsproj) on GitHub.
 
 You have the following final solution layout:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/testing/unit-testing-fsharp-with-dotnet-test.md]: ditto 

```suggestion
```
```

File: docs/core/testing/unit-testing-fsharp-with-mstest.md [hunk 14]
```diff
@@ -17,7 +17,7 @@ Inside this new directory, run [`dotnet new sln`](../tools/dotnet-new.md) to cre
 makes it easier to manage both the class library and the unit test project.
 Inside the solution directory, create a *MathService* directory. The directory and file structure thus far is shown below:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/testing/unit-testing-fsharp-with-mstest.md]:
```suggestion
```
```

File: docs/core/testing/unit-testing-fsharp-with-mstest.md [hunk 15]
```diff
@@ -37,7 +37,7 @@ to add the class library project to the solution.
 
 Next, create the *MathService.Tests* directory. The following outline shows the directory structure:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/testing/unit-testing-fsharp-with-mstest.md]:
```suggestion
```
```

File: docs/core/testing/unit-testing-fsharp-with-mstest.md [hunk 16]
```diff
@@ -58,15 +58,15 @@ Make the *MathService.Tests* directory the current directory and create a new pr
...
 dotnet add reference ../MathService/MathService.fsproj
 ```
 
 You can see the entire file in the [samples repository](https://github.com/dotnet/samples/blob/master/core/getting-started/unit-testing-with-fsharp/MathService.Tests/MathService.Tests.fsproj) on GitHub.
 
 You have the following final solution layout:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/testing/unit-testing-fsharp-with-mstest.md]:
```suggestion
```
```

File: docs/core/testing/unit-testing-fsharp-with-nunit.md [hunk 17]
```diff
@@ -27,7 +27,7 @@ dotnet new sln
 
 Next, create a *MathService* directory. The following outline shows the directory and file structure so far:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/testing/unit-testing-fsharp-with-nunit.md]:
```suggestion
```
```

File: docs/core/testing/unit-testing-fsharp-with-nunit.md [hunk 18]
```diff
@@ -56,7 +56,7 @@ dotnet sln add .\MathService\MathService.fsproj
 
 Next, create the *MathService.Tests* directory. The following outline shows the directory structure:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/testing/unit-testing-fsharp-with-nunit.md]:
```suggestion
```
```

File: docs/core/testing/unit-testing-fsharp-with-nunit.md [hunk 19]
```diff
@@ -91,7 +91,7 @@ You can see the entire file in the [samples repository](https://github.com/dotne
 
 You have the following final solution layout:
 
-```
+```console
```
Comment (mairaw) [inline docs/core/testing/unit-testing-fsharp-with-nunit.md]:
```suggestion
```
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you @v-thpra! Left a few suggestions for you. Feel free to reach out if you have any questions.

File: docs/architecture/microservices/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md (hunk 1 above)
Comment (v-thpra) [reply docs/architecture/microservices/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md]: Good idea! That seems to work and is more accurate.

**Comment by v-thpra** [comment]
@mairaw I sent you an email suggesting we could use "filesystem" as an ID for the directory structures in code blocks. 

However, I now think it would be better to leave them blank - as you suggested - and then review all of the blank ones that are left at the end. 

When we see them all, you can better determine if it's worth creating language IDs for non-language items. Let me know if you agree with that strategy. Thanks!

**Comment by mairaw** [comment]
That sounds good @v-thpra! Hopefully, we won't have many empty ones when you're done.

**Review by mairaw — APPROVED** [review]
LGTM

**Pull request merged**